js/components/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+// import "./App.css";
```

Comment:
What is the reason for leaving several of these commented out lines? Do you anticipating using them in the future or needing them for reference?

Reply:
Excessive caution, mostly -- I didn't want to throw away too much, especially as this is the first commit for a lot of this code, and some lines (such as the schema definition in DragAndDropTable.js) might come in handy. But there's definitely some cruft that doesn't need to be there -- a prime example being references to files that don't exist, like App.css.
I'm happy to do another pass to clean this up a bit.

Reply:
Sounds good. Leave in what you consider appropriate, such as the schema definition, and take out what seems like leftovers we won't need. For basic things like imports and all, know enough that we shouldn't need them for reference.